External Code/ASIHttpRequest/Classes/ASIAuthenticationDialog.m

```diff
@@ -145,7 +145,7 @@ - (void)orientationChanged:(NSNotification *)notification
 	CGRect f = [[UIScreen mainScreen] applicationFrame];
 
 	// Swap the frame height and width if necessary
- 	if (UIDeviceOrientationIsLandscape(o)) {
+ 	if (UIInterfaceOrientationIsPortrait(o)) {
```

Comment:
I don't think it matters, because I suspect this code is never called in our app, but is the behaviour change here intentional?

Reply:
Oops no it was not. Thanks for catching that one. Will update.